qiita_ware/analysis_pipeline.py

```diff
@@ -28,6 +28,9 @@ def _build_analysis_files(analysis, r_depth=None, **kwargs):
     r_depth : int, optional
         Rarefaction depth for biom table creation. Default None
     """
+    if not analysis.jobs:
+        raise RuntimeError("Analysis %d has no jobs attached!" % analysis.id)
```

Comment:
Can you update the docstring to reflect that the function raises an error? Thanks!